cmd/statesync/trust.go

```diff
@@ -0,0 +1,157 @@
+package statesync
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/oasisprotocol/oasis-core/go/consensus/api"
+	"github.com/oasisprotocol/oasis-core/go/consensus/cometbft/config"
+
+	"github.com/oasisprotocol/cli/cmd/common"
+	cliConfig "github.com/oasisprotocol/cli/config"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+	"github.com/spf13/cobra"
+)
+
+const (
+	// maxVerificationTime significantly overestimates the time needed for the
+	// light client to fetch and verify light headers and possibly detect and penalize
+	// misbehavior of byzantine node.
+	maxVerificationTime = 24 * time.Hour
+	// trustPeriodFraction defines the fraction of the debonding period that determines
+	// a valid trust period.
+	//
+	// According to the CometBFT documentation, the sum of the trust period, the time
+	// required to verify headers, and the time needed to detect and penalize misbehavior
+	// should be significantly smaller than the total debonding period.
+	trustPeriodFraction = 3.0 / 4.0
+)
+
+var trustCmd = &cobra.Command{
+	Use:   "trust",
+	Short: "Show the recommended light client trust for consensus state sync",
+	Args:  cobra.NoArgs,
+	RunE: func(_ *cobra.Command, _ []string) error {
+		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+		defer cancel()
+
+		// Establish connection with the target network.
+		cfg := cliConfig.Global()
+		npa := common.GetNPASelection(cfg)
+		conn, err := connection.Connect(ctx, npa.Network)
+		if err != nil {
+			return fmt.Errorf("failed to establish connection with the target network: %w", err)
+		}
+
+		trustCfg, err := calcTrustCfg(ctx, conn)
+		if err != nil {
+			return fmt.Errorf("failed to calculate consensus state sync trust root: %w", err)
+		}
+
+		switch common.OutputFormat() {
+		case common.FormatJSON:
+			str, err := common.PrettyJSONMarshal(trustCfg)
+			if err != nil {
+				return fmt.Errorf("failed to pretty json marshal: %w", err)
+			}
+			fmt.Println(string(str))
+		default:
+			fmt.Println("Trust period: ", trustCfg.Period)
+			fmt.Println("Trust height: ", trustCfg.Height)
+			fmt.Println("Trust hash: ", trustCfg.Hash)
+		}
+
+		return nil
+	},
+}
+
+// calculateTrustCfg calculates and verifies the recommended trust config.
+func calcTrustCfg(ctx context.Context, conn connection.Connection) (config.TrustConfig, error) {
+	var trust config.TrustConfig
+
+	latestBlk, err := conn.Consensus().Core().GetBlock(ctx, api.HeightLatest)
+	if err != nil {
+		return trust, fmt.Errorf("failed to get latest block: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to get latest block: %w", err)
```

I prefer `nil`, as I'm not sure if `trust` has zero value.